pages/security.md

```diff
@@ -183,6 +183,64 @@ heroku config:set \
   SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_OIDC_CLIENT_SECRET="$SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_OIDC_CLIENT_SECRET"
 ```
 
+#### Create a Native Okta App for Mobile
+
+If you're developing a mobile app with JHipster's [Ionic](https://github.com/jhipster/generator-jhipster-ionic) or [React Native](https://github.com/jhipster/generator-jhipster-react-native) blueprints, you will need to create a native app on Okta if you're using OIDC. 
+
+Using the [Okta CLI](https://cli.okta.com), run `okta apps create`. Select the default app name, or change it as you see fit. Choose **Native** and press **Enter**.
+
+- **Ionic**: Change the Redirect URI to `[http://localhost:8100/callback,dev.localhost.ionic:/callback]` and the Logout Redirect URI to `[http://localhost:8100/logout,dev.localhost.ionic:/logout]`.
+- **React Native**: Use `[http://localhost:19006/,https://auth.expo.io/@<username>/<appname>]` for redirect URIs. 
+
+**NOTE:** `dev.localhost.ionic` is the default scheme, but you can also use something more traditional like `com.okta.dev-133337` (where `dev-133337.okta.com` is your Okta Org URL). If you change it, be sure to update the `scheme` in your Ionic app's `src/environments/environment.ts` and the redirect URLs in `src/app/auth/factories/auth.factory.ts`.
+
+The Okta CLI will create an OIDC App in your Okta Org. It will add the redirect URIs you specified and grant access to the Everyone group.
+
+```shell
+Okta application configuration:
+Issuer:    https://dev-133337.okta.com/oauth2/default
+Client ID: 0oab8eb55Kb9jdMIr5d6
+```
+
+**NOTE**: You can also use the Okta Admin Console to create your app. See [Create a Native App](https://developer.okta.com/docs/guides/sign-into-mobile-app/create-okta-application/) for more information.
+
+#### Add Claims to Access Token
+
+In order to authentication successfully with your mobile app, you have to do a bit more configuration in Okta. Since the mobile client will only send an access token to JHipster, you need to 1) add a `groups` claim to the access token and 2) add a couple more claims so the user's name will be available in JHipster.
```

Comment:
```suggestion
In order to authenticate successfully with your mobile app, you have to do a bit more configuration in Okta. Since the mobile client will only send an access token to JHipster, you need to 1) add a `groups` claim to the access token and 2) add a couple more claims so the user's name will be available in JHipster.
```